apps/src/studio/studio.js

```diff
@@ -1093,9 +1093,11 @@ Studio.init = function(config) {
   // pre-load images asynchronously
   // (to reduce the likelihood that there is a delay when images
   //  are changed at runtime)
-  preloadActorImages();
-  preloadProjectileImages();
-  preloadBackgroundImages();
+  if (config.skin.id == "studio") {
```

Comment:
Some of current preloading is hardcoded to original studio assets.. the projectile list is using

  'blue_fireball',
  'purple_fireball',
  'red_fireball',
  'purple_hearts',
  'red_hearts',
  'yellow_hearts'

and the background images are the K1 images.

And given our thread with Chris it sounds like there's a question around whether we should still preload given the size of the assets.  I'm happy to get it working for Infinity, but preferred to get this change in first and then see whether we still need preloading for the new assets, while leaving original studio skin behaviour intact.